.eslintrc.js

```diff
@@ -32,6 +33,7 @@ module.exports = {
 			},
 		],
 		'jsx-a11y/href-no-hash': 'off',
+		'import/no-unresolved': 'off',
```

Comment:
This is not needed if you fix the import problems in pages/apa (index.tsx, pathogens/[pathogen-id].tsx and pathogens/index.tsx)

Reply:
Resolved